Update changelog with new features and updates (#4616)
timescale/docs: about/changelog.md

All the latest features and updates to $CLOUD_LONG.

## 🧭  Activity log and TimescaleDB v2.24
<Label type="date">December 12, 2025</Label>

### Activity log

> Tiger Cloud now offers an activity log accessible from your list of all services. The log serves as a record of actions that have happened to your services and Tiger Cloud account, such as service resizes and project invitations. The log includes the corresponding service (where applicable), the user who performed the action, and a description of the action itself. You can  suggest new actions to record on the `Activity` tab.

Reviewer: ```suggestion
Tiger Cloud Console now offers the Activity tab that displays the activity log for all your services. This serves as a record of actions that have happened to your services and Tiger Cloud account, such as service resizes and project invitations. The activity log includes the corresponding service (where applicable), the user who performed the action, and a description of the action itself. You can  suggest new actions to record on the `Activity` tab.
```

![Activity log on Tiger Cloud](https://assets.timescale.com/docs/images/tiger-cloud-console/activity-log.png)

### TimescaleDB 2.24
TimescaleDB 2.24 was released on December 3rd and is now available to all users on Tiger Cloud. TimescaleDB 2.24 delivers more efficient recompression operations, expanded use of continuous aggregates, and better invalidation behavior. It also brings back support for bloom filters on ARM-based architectures. 

#### Highlighted features in TimescaleDB 2.24

**In-memory recompression**

A new `recompress := true` option for `convert_to_columnstore()` performs batch compaction entirely in memory. 

For example: `CALL convert_to_columnstore('<chunk_name>', recompress := true);`

This approach is 4–5 times faster than the previous spill-to-disk method, and reduces I/O for workloads with many small or uneven batches. This can be helpful if you are ingesting unordered data via direct compress and need to optimize your batches, or when you add new sparse indexes and need to build them on existing chunks.

> **Bloom filters on Tiger Cloud (ARM-based services)**

Reviewer: ```suggestion
**Bloom filters on ARM-based Tiger Cloud services**
```

> On Tiger Cloud (ARM-based services), bloom filters return with corrected hashing support for ARM architecture. A misconfigured hashing library previously required disabling bloom indexes on ARM-based services. This release restores bloom filter functionality with a new index version.

Reviewer: ```suggestion
On ARM-based Tiger Cloud services, bloom filters return with corrected hashing support for the ARM architecture. A misconfigured hashing library previously required disabling bloom indexes on ARM-based services. This release restores bloom filter functionality with a new index version.
```

> For self-hosted TimescaleDB users, nothing changes if you're using an AMD64 architecture. Otherwise, please [see here](https://github.com/timescale/timescaledb/pull/8761#user-content-changelog). For Tiger Cloud customers, recompression is only required for services that want to rebuild bloom filters on existing chunks. New chunks receive bloom indexes automatically.

Reviewer: ```suggestion
For self-hosted TimescaleDB users, nothing changes if you're using an AMD64 architecture. Otherwise, please [see here](https://github.com/timescale/timescaledb/pull/8761#user-content-changelog). For Tiger Cloud customers, recompression is only required for services where you want to rebuild bloom filters on existing chunks. New chunks receive bloom indexes automatically.
```
